app/channels/application_cable/connection.rb

```diff
@@ -1,4 +1,36 @@
 module ApplicationCable
   class Connection < ActionCable::Connection::Base
+    identified_by :current_user, :onsite
+
+    def connect
+      # Identify if the user is 'onsite' so that pre-rendered html can be sent
+      self.current_user, self.onsite = find_verified_user
+      log_tag = current_user.try(:name) || SecureRandom.hex
```

Comment:
For convention's sake, I've been using `SecureRandom.uuid` when needing a random identifier like this.